vizro-core/pyproject.toml

```diff
@@ -83,4 +83,4 @@ filterwarnings = [
   "ignore:Custom format string detected."
 ]
 norecursedirs = ["tests/tests_utils", "tests/js"]
-pythonpath = ["tests/tests_utils", "examples/kpi"]
+pythonpath = ["tests/tests_utils", "examples/kpi", "examples/_chart-gallery"]
```

Comment:
@l0uden - we have a similar issue here as with the KPI dashboard (modules not being found in the integration tests etc.). Take a look at the KPI PR for reference, where you have pushed this fix: https://github.com/mckinsey/vizro/pull/505

I just saw that @antonymilne commented we shouldn't fix it this way: https://github.com/mckinsey/vizro/pull/505#discussion_r1625922228

Anyway, the solution that worked last time doesn't work here anymore as we otherwise get different error messages regarding modules. What's the best way to solve this right now? @l0uden - could you take a look? 🤔

Reply:
I can take a look at this, but just to check I understand: the problem here just is what happens when you run `hatch run test-integration`, right?

Reply:
This isn't anything specific to `dash_duo` - it's just because the `runpy.run_path` function doesn't add the cwd to the `sys.path` so the behaviour when you're running the app is slightly different to when you do `python app.py` (which is what it's meant to emulate).

So adding the directory to `pythonpath` is basically the right solution but a bit too heavy-handed/disruptive since it's a global change. 

I just pushed [aa9931a](https://github.com/mckinsey/vizro/pull/559/commits/aa9931a2faa368e149cb2fee7c4d00277aea9b66) which is my preferred fix here - let's see if it passes.

Reply:
It didn't... but hopefully the commits pushed just now will. 🤞

Reply:
Just saw that it worked! 🥳 💯 🚀